UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1319,13 +1319,14 @@ private void AddToSurface(ReagentMix Chemicals, ReagentMix mix)
 		[Server]
 		public virtual void OnGib()
 		{
-			_ = SoundManager.PlayAtPosition(CommonSounds.Instance.Slip, gameObject.transform.position,
+			SoundManager.PlayAtPosition(CommonSounds.Instance.Slip, gameObject.transform.position,
```

Comment:
_ = isn't this need because it tries returning a task?

Reply:
`_ =` <- this just stops the rest of the code from running until the sound playing is done. It's not needed.

Reply:
no. ``await`` would do that. By doing ``_ =`` all you are doing is discarding the return from the method call and avoiding a compiler warning

Reply:
alr, brought it back